coniferest/calc_paths_sum.pxd

```diff
@@ -11,6 +11,8 @@ cdef packed struct selector_t:
     np.float64_t value
     # Index of right subtree
     np.int32_t right
+    # Log of samples count covered by this node
+    np.float32_t log_n_node_samples
```

Comment:
Is it a right trade-of between building and scoring performance?

Reply:
I hope so